src/app/pages/properties/properties.component.ts

```diff
@@ -28,13 +29,18 @@ export class PropertiesComponent implements OnInit {
     private propertiesServices: PropertiesService,
     private spinner: NgxSpinnerService,
     private toastrService: ToastrService,
-    private router: Router
+    private router: Router,
+    private titleService: Title
   ) { }
 
   ngOnInit(): void {
     this.setProperties(this.propertiesUrl);
   }
 
+  setDocTitle(title: string) {
+    console.log('current title:::::' + this.titleService.getTitle());
```

Comment:
Good job working on this feature @dorothyas I'm able to view property details successfully however I believe we can do away with `console.log`.

Reply:
Thanks @Tevinthuku. All done